bm-instance-agent/bm_instance_agent/systems/linux/driver.py

```
@@ -150,12 +150,13 @@ def discovery_target(self, instance_obj):
         if not stderr:
             LOG.info("iscsi target:%s has logged" % target_name)
+            rescan_sids_target_name(target_name)
             return
 
         discovery_cmd = 'iscsiadm -m discovery -t sendtargets -p {address}:{port}'.format(
-                                    address=instance_obj.gateway_ip,
-                                    port=3260,
-                                    target_name=target_name)
+            address=instance_obj.gateway_ip,
+            port=3260,
+            target_name=target_name)
```

Ruff (0.14.11)

`.format` call has unused named argument(s): target_name

Remove extra named arguments: target_name

(F522)